src/Components/SignInPage/SignInPage.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import './SignInPage.css';
+import { Link } from 'react-router-dom';
+
+class SignIn extends Component {
```

Comment:
Does this need to be a class component if it doesn't hold any state? Would a functional component work here?